packages/lit-dev-tools-esm/src/fake-github-server.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import Koa from 'koa';
+import {fakeGitHubMiddleware} from './fake-github-middleware.js';
+import {getEnvironment} from 'lit-dev-tools-cjs/lib/lit-dev-environments.js';
+
+const ENV = getEnvironment();
```

Comment:
NIT: Is there a way to assert which variant of environment this is to avoid the non null operator?

Reply:
Updated the directly exported environments to retain their narrower types, using a function that just enforces they are subtypes of LitDevEnvironment. Now when you directly import one, it knows which files must be defined.

Though, it's actually only the `dev-server.ts` file that imports a specific environment, not this one. So I moved the `envRequired` function to this file, since it's no longer needed in `dev-server.ts`.